Source/Core/DolphinWX/Config/AudioConfigPane.cpp

```diff
@@ -55,50 +56,54 @@ void AudioConfigPane::InitializeGUI()
   m_dpl2_decoder_checkbox->SetToolTip(
       _("Enables Dolby Pro Logic II emulation using 5.1 surround. Certain backends only."));
 
-  const int space5 = FromDIP(5);
+  m_soundtouch_timestretch_checkbox->SetToolTip(_("Enables timestretching support, smoother audio "
+                                                  "but can add latency. Certain backends only."));
+  const int space6 = FromDIP(6);
```

Comment:
This was my first time dealing with WX, so something is bound to be wrong... Reverted, as at first I thought it was causing compilation errors but now I see that it was another problem.